infra/modules/providers/azure/app-monitoring/README.md

```diff
@@ -0,0 +1,117 @@
+
+Terraform resources used to define the `app-monitoring` module include the following:
+
+- [azurerm_monitor_metric_alert](https://www.terraform.io/docs/providers/azurerm/r/monitor_metric_alert.html)
+
+- [azurerm_monitor_action_group](https://www.terraform.io/docs/providers/azurerm/r/monitor_action_group.html)
+
+Cobalt modules used to deploy the `app-monitoring` module include the following:
+
+- [Service Plan Module](../service-plan)
+
+Variables used to define and deploy the `app-monitoring` module:
+
+| Required Variables | Required | Scope | Description |
+|-------------|-----------|-----------|-----------|
+| `resource_group_name` | &check;| Template | The name of the resource group containing deployed resources. |
+| `resource_ids` | &check; | Template  | Resource Ids to be monitored. Value can either be a list of VM Ids or a single chosen Azure Resource Id. See [Monitoring at scale](https://docs.microsoft.com/en-us/azure/azure-monitor/platform/alerts-metric-overview#monitoring-at-scale-using-metric-alerts-in-azure-monitor) for more information choosing multiple resources.|
+| `action_group_name` | &check; | Template  | The name of the action group. |
+| `action_group_short_name` | &check;  | Module | The abbreviated name of the action group. |
+| `action_group_email_receiver` | &check;  | Template  | The e-mail address for an action group. |
+| `action_group_email_receiver_name` | &check;  | Module | The display name of the action group's e-mail address. |
+| `metric_alert_criteria_namespace` | &check; | Template  | A monitorable resource namespace with configurable alert criteria. |
+| `metric_alert_name` | &check; | Template  | The display name of a group of metric alert criteria. |
+| `metric_alert_criteria_name` | &check; | Template | A predefined Azure resource alert monitoring rule name. |
+| `metric_alert_criteria_aggregation` | &check; | Template  | The calculation used for building metric alert criteria. |
+| `metric_alert_criteria_operator` | &check; | Template  | A logical operator used for building metric alert criteria. |
+| `metric_alert_criteria_threshold` | &check; | Template | The criteria threshold value that activates the metric alert. |
+| `scaling_name` | &check; | Module | Local var with a constant value of "Instance". |
+| `scaling_operator` | &check; | Module | Local var with a constant value of "Include". |
+| `scaling_values` | &check; | Template | App instance names made available from app service plan scaling options. |
+
+## Module Usage
+
+The `action group` variables are only configurable for sending e-mail notifications when `metric alert criteria` crosses a configured threshold value. The values used to define `metric alert criteria` are provided by a template. If a template does not provide an e-mail by satisfying the `action_group_email_receiver` variable, all other configuration values will be ignored and this module will not be present in the Terraform plan generated for deployment:
```

Comment:
Providing an example to the `module "app_monitoring"` definition should be sufficient here.

Reply:
This was removed in favor of the module "app_monitoring" definition in the form of a code block.